core/Pos/Aeson/Core.hs

```diff
@@ -0,0 +1,61 @@
+-- | Aeson instances for core
+
+module Pos.Aeson.Core
+       (
+       ) where
+
+import           Universum
+
+import           Data.Aeson             (FromJSON (..), ToJSON (toJSON))
+import           Data.Aeson.TH          (defaultOptions, deriveFromJSON, deriveJSON,
+                                         deriveToJSON)
+import           Data.Time.Units        (Microsecond, Millisecond)
+import qualified Serokell.Aeson.Options as S (defaultOptions)
+import           Serokell.Util.Base64   (JsonByteString (..))
+
+import           Pos.Aeson.Crypto       ()
+import           Pos.Aeson.Fee          ()
+import           Pos.Binary.Class       (AsBinary (..))
+import           Pos.Core.Types         (ApplicationName (..), BlockCount (..),
+                                         BlockVersionData, ChainDifficulty, Coin,
+                                         CoinPortion, EpochIndex (..), LocalSlotIndex,
+                                         SharedSeed (..), SlotCount (..), SlotId,
+                                         SoftforkRule, Timestamp,
+                                         unsafeCoinPortionFromDouble)
+import           Pos.Core.Vss           (VssCertificate)
+
+instance ToJSON SharedSeed where
+    toJSON = toJSON . JsonByteString . getSharedSeed
+
+instance FromJSON SharedSeed where
+    parseJSON v = SharedSeed . getJsonByteString <$> parseJSON v
+
+instance FromJSON (AsBinary w) where
+    parseJSON v = AsBinary . getJsonByteString <$> parseJSON v
+
+deriving instance ToJSON SlotCount
+
+instance FromJSON CoinPortion where
+    parseJSON v = unsafeCoinPortionFromDouble <$> parseJSON v
+
+deriveJSON defaultOptions ''Coin
```

Comment:
It should also be below `NOTE` (it's used by frontend).
Also I will be glad if you rebase and make all aeson instances changes in one commit, because it's hard to understand what happened with them in this PR.